Deprecate setting toolchain properties with a project property (#34574)
gradle/gradle: platforms/documentation/docs/src/docs/userguide/releases/upgrading/upgrading_version_9.adoc

=====
====

[[toolchain-project-properties]]

> ==== Specifying toolchain configuration using project properties is now deprecated

Reviewer: ```suggestion
==== Deprecation of project properties for toolchain configuration

```

> In previous versions of Gradle, it was possible to specify toolchain configuration with _project properties_ on the command line using `-P`.

Reviewer: ```suggestion
In previous versions of Gradle, you could configure toolchains using [project properties](userguide/build_environment.html#sec:project_properties) on the command line with the `-P` flag.
```

> For example, to enable auto-detection of toolchains, you could specify `-Porg.gradle.java.installations.auto-detect=false`.

Reviewer: ```suggestion
For example, to enable toolchain auto-detection, you could use `-Porg.gradle.java.installations.auto-detect=false`.
```

> This behavior is now deprecated and will be removed in Gradle 10.0.0.

Reviewer: ```suggestion
This behavior is deprecated and will be removed in Gradle 10.0.0.
```

> Instead, these properties should be specified as Gradle properties on the command line using `-D`, for example `-Dorg.gradle.java.installations.auto-detect=false`.

Reviewer: ```suggestion
Instead, you should specify these settings as [Gradle properties](userguide/build_environment.html#sec:gradle_configuration_properties) using the `-D` flag:
```

> 

Reviewer: ```suggestion

```text
-Dorg.gradle.java.installations.auto-detect=false
//```
```
